lib/capistrano/configuration.rb

```diff
@@ -27,6 +27,10 @@ def set(key, value)
       config[key] = value
     end
 
+    def append(key, value)
+      set(key, fetch(key, []) << value)
```

Comment:
@Kriechi ping I have some time for Capistrano releases today, if you can update #1447 I can get this out in 3.5.0 today.